### PR TITLE
updating the home page (and some small changes to surplus and produce)

### PR DESCRIPTION
**Pull request by andrewhojel**
The home page is updated to deal with all the new pages (that have been merged in recent PRs), localstorage is added to maintain states after page reload, and a selected class is added to identify which page is selected

The changes to surplus and produce are very minimal (small oversights of formatting)

### CODE REVIEW

File: view/src/pages/home.js [hunk 1]
```diff
@@ -166,20 +276,92 @@ class Home extends Component {
...
+                  />
                 </ListItemIcon>
-                <ListItemText primary="Todo" />
+                <ListItemText primary="Farms" />
               </ListItem>
-
-              <ListItem button key="Account" onClick={this.loadAccountPage}>
+              <ListItem
+                className={this.isSelected("foodbanks") && classes.selected}
+                button
```
Comment (jessimb) [inline view/src/pages/home.js:294]: nit: swap the ordering (to match the other ListItems)

File: view/src/pages/home.js [hunk 2]
```diff
@@ -166,20 +276,92 @@ class Home extends Component {
...
+                className={this.isSelected("surplus") && classes.selected}
+                key="Surplus"
+                onClick={this.loadSurplusPage}
+              >
                 <ListItemIcon>
-                  <AccountBoxIcon />
+                  <ChartIcon
+                    className={
+                      this.isSelected("surplus") && classes.selectedIcon
+                    }
```
Comment (jessimb) [inline view/src/pages/home.js:330]: i think this should fit on one line

Comment (jessimb) [reply view/src/pages/home.js:330]: same goes for the other instances where the className was broken into several lines. not sure if this is prettier helping or copy/paste

File: view/src/pages/home.js [hunk 3]
```diff
@@ -189,7 +371,30 @@ class Home extends Component {
...
+              <Account alert={this.alert} />
+            ) : this.state.toRender === "produce" ? (
+              <Produce alert={this.alert} />
+            ) : this.state.toRender === "surplus" ? (
+              <Surplus alert={this.alert} main={true} />
+            ) : this.state.toRender === "farms" ? (
+              <Farms alert={this.alert} />
+            ) : (
+              <FoodBanks alert={this.alert} main={true} />
+            )}
```
Comment (jessimb) [inline view/src/pages/home.js:396]: I wonder if a switch statement would be more readable? I'm not used to seeing the ternary operator chained like this. 

I'll admit I'm biased against ternary, even when it's the better choice. So I could certainly be wrong here.

File: view/src/pages/home.js (hunk 2 above)
Comment (andrewhojel) [reply view/src/pages/home.js:330]: all the weird line breaks are from prettier

File: view/src/pages/home.js [hunk 4]
```diff
@@ -132,44 +132,13 @@ class Home extends Component {
...
-  };
-
-  /** Loads the food banks page (and stores in local storage to remember after page reload) */
-  loadFoodBanksPage = (event) => {
-    localStorage.setItem("toRender", "foodbanks");
-    this.setState({ toRender: "foodbanks" });
+  /** Loads the page with the provided name (and stores it in local storage to remember post page reload) */
+  loadPage = (name) => {
+    localStorage.setItem("toRender", name);
+    this.setState({ toRender: name });
```
Comment (jessimb) [inline view/src/pages/home.js:138]: Much cleaner!

**Pull request merged**